static/app/components/charts/optionCheckboxSelector.tsx

```diff
@@ -19,31 +19,37 @@ const defaultProps = {
   menuWidth: 'auto',
 };
 
-type Props = {
+export type OptionCheckboxSelectorProps = {
   onChange: (value: string[]) => void;
   options: (SelectValue<string> & {checkboxHidden?: boolean})[];
   selected: string[];
   title: string;
 } & typeof defaultProps;
 
-type State = {
+export type OptionCheckboxSelectorState = {
```

Comment:
oof, even more annoying is exporting state, something that should be totally internal 😬  can we avoid this?

Reply:
agree... we shouldn't export `state`

Reply:
Yeah, I can drop this. I was trying to see why mountWithTheme wasnt inferring the correct component state... calling wrapper.props() should give us the correct prop type.